assets/main.scss

```diff
@@ -28,20 +28,25 @@
 .language-python + .language-plaintext code { background-color: white !important; }
 .language-python + .language-plaintext pre  { background-color: white !important; }
 
-// for Jupyter Notebook HTML Code Cells from https://www.fast.ai/public/css/hyde.css
+// for Jupyter Notebook HTML Code Cells modified from https://www.fast.ai/public/css/hyde.css
 
 .input_area pre, .input_area div {
-        margin-bottom:0.1rem !important;
-        margin-top:0.1rem !important;
+        margin-bottom:2rem !important;
+        margin-top:1.5rem !important;
         padding-bottom:0 !important;
         padding-top:0 !important;
+        background: #323443 !important;
+        -webkit-font-smoothing: antialiased;
+        text-rendering: optimizeLegibility;
+        font-family: BlinkMacSystemFont,'Segoe UI',Roboto,Ubuntu,'Helvetica Neue', sans-serif;
+        font-family: Menlo, Monaco, "Courier New", monospace;
```

Comment:
I think you need less ugly Windows fonts in the list, ahead of Courier! :) e.g. Consolas and Lucida Console

Reply:
Thank you so much, this is something I really struggle with and really appreciate the suggestions!   When it comes to aesthetics I am very "challenged" 😢  Will make the change 🙇